src/carmine_sentinel/core.clj

```diff
@@ -132,24 +134,26 @@
       true)))
 
 (defn- pick-specs-from-sentinel-raw-states [raw-states]
-  (->> raw-states
-       (map (partial apply hash-map))
-       (map (fn [{:strs [ip port]}]
-              {:host ip
-               :port (Integer/valueOf ^String port)}))))
+  (map
```

Comment:
Hi ,i think thread macros are more readable than nested function invocation, and it's a recommend code style in clojure.I found that your modifictions have removed all thread macros, it's not a good practice.

Reply:
I don't mind bringing the threaded macros back, I found debugging them a bit painful when investigating the code.
Any other issue besides the macros?